pkg/firewalld/firewalld.go

```diff
@@ -0,0 +1,96 @@
+package firewalld
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/godbus/dbus"
+)
+
+const (
+	fInterface = "org.fedoraproject.FirewallD1"
+	fObjPath   = "/org/fedoraproject/FirewallD1"
+)
+
+// Protocol type represents the type of protocol
+type Protocol string
+
+// Supported protocol types
+const (
+	ProtoTCP         Protocol = "tcp"
+	ProtoUDP         Protocol = "udp"
+	dbusSignalChSize          = 10
```

Comment:
where  `dbusSignalChSize` is used, am not able to find reference to it

Reply:
This was intended to be used in reconcileFirewalld(). Will fix that.